brian2/stateupdaters/base.py

```diff
@@ -127,10 +138,16 @@ def apply_stateupdater(equations, variables, method):
             if the_method is None:
                 raise ValueError(('No stateupdater that is suitable for the '
                                   'given equations has been found.'))
-            logger.info('Using state updater: %r' % the_method)
+            if group_name is not None:
+                msg_text = ("No numerical integration method specified for group "
+                            "'{group_name}', choosing method '{method}'.")
+            else:
+                msg_text = ("No numerical integration method specified, "
+                            "choosing method '{method}'.")
+            logger.info(msg_text.format(group_name=group_name,
+                                        method=the_method), 'method_choice')
```

Comment:
Yes, the idea was that it is a hint about the time it takes. Just adding the time is a good idea (I would probably not bother with a threshold). There is a subtle issue that the full time this takes is a mix of two things: discarding non-applicable methods and applying the methods. I think both informations are interesting, but it is probably a bit too verbose for `INFO` -- so I think I'll just add it to `DEBUG`?